lib/util.js

```diff
@@ -573,7 +585,8 @@ exports.isPrimitive = isPrimitive;
 function isBuffer(arg) {
   return arg instanceof Buffer;
 }
-exports.isBuffer = isBuffer;
+exports.isBuffer = deprecate(isBuffer,
+    'util.isBuffer is deprecated, please use a user-land alternative.');
```

Comment:
They are exactly the same.
On Thu, Nov 12, 2015 at 4:38 AM hellboy81 <notifications@github.com> wrote:

> In lib/util.js
> <https://github.com/nodejs/node/pull/1301#discussion_r44651882>:
>
> > @@ -573,7 +585,8 @@ exports.isPrimitive = isPrimitive;
> >  function isBuffer(arg) {
> >    return arg instanceof Buffer;
> >  }
> > -exports.isBuffer = isBuffer;
> > +exports.isBuffer = deprecate(isBuffer,
> > +    'util.isBuffer is deprecated, please use a user-land alternative.');
>
> And isBuffer is better than instanceof Buffer ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/1301/files#r44651882>.
>